scripts/scrape.py

```diff
@@ -0,0 +1,25 @@
+import argparse
+import os
+import sys
+sys.path.append('src')  # if there is a better way, please tell me
```

Comment:
you don't need this. You can instead import the other files as `import src.wiktionary_scraper as ws` for instance

Reply:
I tried and that doesn't work, even though according to the official doc it should...

Reply:
are you running your scripts from the root of your project ?

Reply:
yes

Reply:
let me see...

Reply:
I know. this is because of the way you run your code... it should be
`python3 -m scripts.scrape`. Let me know if it helps

Reply:
It works \o/